docs/topics/readme/zip-rest-http-secured-spring-boot.properties

```diff
@@ -2,5 +2,5 @@ localRunCMD=mvn clean fabric8:deploy -Popenshift -DskipTests -DSSO_AUTH_SERVER_U
 app-name=sb
 gitCMD=git clone git@github.com:${loggedUser}/${targetRepository}
 guideURL=http://launcher.fabric8.io/docs/spring-boot-runtime.html
-prerequisite=IMPORTANT: This booster requires Java 8 JDK or greater and Maven 3.3.x or greater.
-integrationTestSection=To run the integration test on the Secured HTTP API booster follow the steps in the link:https://launcher.fabric8.io/docs/spring-boot-runtime.html#running-the-secured-booster-integration-tests_context[Secured Booster documentation].
+prerequisite=IMPORTANT: This example application requires Java 8 JDK or greater and Maven 3.3.x or greater.
+integrationTestSection=To run the integration test on the Secured HTTP API example application follow the steps in the link:https://launcher.fabric8.io/docs/spring-boot-runtime.html#running-the-secured-booster-integration-tests_context[Secured Booster documentation].
```

Comment:
To run the integration test on the Secured HTTP API example application follow--> Need a comma between application and follow
`To run the integration test on the Secured HTTP API example application, follow`
There are a lot of instances with the same statement. Consider making this update in all instances for better clarity.